Library/Formula/duplicity.rb

```diff
@@ -319,6 +319,8 @@ def install
       r.stage { system "python", *Language::Python.setup_install_args(libexec/"vendor") }
     end
 
+    chmod 0644, Dir[vendor_site_packages/"prettytable-*-py*.egg-info/*"]
```

Comment:
Is this a problem for the `pip install`'d version as well? If so, has it been reported upstream?